meta-rpi-domd/recipes-bsp/u-boot/u-boot_2024.04.bb

```diff
@@ -0,0 +1,10 @@
+# require ../../../../poky/meta/recipes-bsp/u-boot/u-boot.inc
```

Comment:
so u-boot_2024.04.bb or u-boot_%.bbappend?

Reply:
Both of them.
u-boot_2024.04.bb for u-boot itself u-boot_%.bbappend has rpi5 specific overrides